lib/src/super_sliver_list.dart

```diff
@@ -270,6 +280,12 @@ class ListController extends ChangeNotifier {
     if (_delegate == delegate) {
       _delegate?.removeListener(notifyListeners);
       _delegate = null;
+      // because list can be modified from callback that can be called from
+      // [AnimateToItem.dispose] we must iterate over copy
+      for (final controller in _runningAnimations.toList()) {
```

Comment:
If `_runningAnimations` is already typed as a list, is it necessary to call the toList conversion method?

```suggestion
      for (final controller in _runningAnimations) {
```

Reply:
Yes, .toList uses List.of to internally which copies the list on VM side. Well, realization is really not important as per se, contract of .toList explicitly tells that it creates a new list.

Reply:
Alright, got it! I think I read it with the browser's width reduced, since your comment above explains the need for `toList` copy. 

Sorry for my mistake!